src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -36,31 +36,28 @@ public NodeAutoCompleteSearchControl()
             InitializeComponent();
             if (Application.Current != null)
             {
-                Application.Current.Deactivated += currentApplicationDeactivated;
+                Application.Current.Deactivated += CurrentApplicationDeactivated;
+                Application.Current.MainWindow.Closing += NodeAutoCompleteSearchControl_Unloaded;
```

Comment:
so maybe it's unimportant, but in the unloaded handler, it would be nice to unsub the unloaded event, so we don't need to wonder if that causes memory leaks.

Reply:
Make sense, added